projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/logins-blocked-status.tsx

```diff
@@ -0,0 +1,130 @@
+import { __ } from '@wordpress/i18n';
+import { useMemo } from 'react';
+import useProduct from '../../../data/products/use-product';
+import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
+import useMyJetpackConnection from '../../../hooks/use-my-jetpack-connection';
+import ShieldOff from './assets/shield-off.svg';
+import { InfoTooltip } from './info-tooltip';
+import { useProtectTooltipCopy } from './use-protect-tooltip-copy';
+import type { ReactElement, PropsWithChildren } from 'react';
+
+export const LoginsBlockedStatus = () => {
+	const slug = 'protect';
+	const { detail } = useProduct( slug );
+	const { isPluginActive: isProtectPluginActive = false } = detail || {};
+	const { isSiteConnected } = useMyJetpackConnection();
+	const {
+		protect: { wafConfig: wafData },
+		plugins,
+	} = getMyJetpackWindowInitialState();
+	const { blocked_logins: blockedLoginsCount, brute_force_protection: hasBruteForceProtection } =
+		wafData;
+
+	const isJetpackPluginActive = useMemo( () => {
+		const jetpackPlugin = Object.values( plugins ).find( plugin => plugin?.Name === 'Jetpack' );
+		return jetpackPlugin && jetpackPlugin.active;
+	}, [ plugins ] );
+
+	// The Brute Force Protection module is available when either the Jetpack plugin Or the Protect plugin is active.
+	const isPluginActive = isProtectPluginActive || isJetpackPluginActive;
+
+	if ( isPluginActive && isSiteConnected ) {
+		if ( hasBruteForceProtection ) {
+			return <BlockedStatus status="success" blockedLoginsCount={ blockedLoginsCount } />;
+		}
+
+		return <BlockedStatus status="inactive" blockedLoginsCount={ blockedLoginsCount } />;
+	}
+	if ( isSiteConnected && blockedLoginsCount > 0 ) {
+		// logins have been blocked previoulsy, but either the Jetpack or Protect plugin is not active
+		return <BlockedStatus status="inactive" blockedLoginsCount={ blockedLoginsCount } />;
+	}
+	return <BlockedStatus status="off" blockedLoginsCount={ blockedLoginsCount } />;
+};
+
+/**
+ * BlockedStatus component
+ *
+ * @param {PropsWithChildren} props - The component props
+ * @param {'success' | 'inactive' | 'off'} props.status - The status of Brute Force Protection
+ * @param {number} props.blockedLoginsCount - The number of blocked login attempts
+ *
+ * @returns {ReactElement} rendered component
+ */
+function BlockedStatus( {
+	status,
+	blockedLoginsCount,
+}: {
+	status: 'success' | 'inactive' | 'off';
+	blockedLoginsCount: number;
+} ) {
+	const tooltipContent = useProtectTooltipCopy();
+	const { blockedLoginsTooltip } = tooltipContent;
+
+	if ( status === 'success' ) {
+		return blockedLoginsCount > 0 ? (
+			<div className="logins_blocked__count">{ blockedLoginsCount }</div>
+		) : (
+			<>
+				<div className="logins_blocked__count">0</div>
+				<InfoTooltip
+					tracksEventName={ 'protect_card_tooltip_open' }
+					tracksEventProps={ {
+						location: 'blocked-logins',
+						status: 'success',
```

Comment:
If you do update the status, we'll want to do that here as well. Also we could probably just replace the hardcoded value here with the variable itself 😄

Reply:
```suggestion
						status,
```